rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -368,7 +368,7 @@ def query_single_address_withdrawals(self, address: ChecksumEvmAddress, to_ts: T
             log.error(f'Failed to query ethereum withdrawals for {address} through etherscan due to {e}. Will try blockscout.')  # noqa: E501
             try:
                 self.ethereum.blockscout.query_withdrawals(address)
-            except (DeserializationError, RemoteError) as othere:
+            except (DeserializationError, RemoteError, KeyError) as othere:
```

Comment:
you need special handling here for `othere` if it's a `KeyError`. You remember that str(KeyError)` is just the missing key. Do what we do in other places.